packages/labs/gen-wrapper-vue/src/lib/wrapper-module-template-sfc.ts

```diff
@@ -89,6 +106,32 @@ const getElementTypeImports = (declaration: LitElementDeclaration) => {
 const getElementTypeExportsFromImports = (imports: string) =>
   imports.replace(/(?:^import)/gm, 'export type');
 
+const renderPropsParam = (
+  reactiveProperties: Map<string, ModelProperty>,
+  events: Map<string, EventModel>
+) => {
+  const hasProps = reactiveProperties.size > 0;
+  if (hasProps) {
+    return javascript`
+      const eventProps = ${renderEvents(events)};
+
+      const props = eventProps as (typeof eventProps & Props);
```

Comment:
It might be possible have no reactive props but non-zero events, in which case we want to keep this part.

Conditions here getting a bit more gnarly here, but perhaps we should also do a `hasEvents = events.size > 0` and account for that too?

`hasPros && hasEvents` is as is.

!hasProps && hasEvents` should still include
```js
const eventProps = ${renderEvents(events)};
const props = eventProps as (typeof eventProps & Props);
```
and `!hasProps && !hasEvents` can be the empty object.

Reply:
can you re-review? it may be simpler than that

i just unconditionally output `eventProps` and `props` now, since they will be empty objects anyway. and only output the loop through vue props when we have reactive props

Reply:
yes doing it binary like that to unconditionally output `eventProps` seems fine. right now i'm only seeing an empty `props` though, unless i'm looking in the wrong place?
https://github.com/lit/lit/blob/4d229f29ee61efb7ab51a31699edab3208782072/packages/labs/gen-wrapper-vue/src/lib/wrapper-module-template-sfc.ts#L130-L132

Reply:
sorry forgot a `git push`! should be there now

Reply:
I see it, thank you!